src/project-2.js

```diff
@@ -44,63 +56,69 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num < 2) return false;
+
+  for (let i = 2; i < num; i++) {
+    if (num % i === 0) return false;
+  }
+  return true;
 };
 
-const returnFirst = (arr) => {
+const returnFirst = arr => arr[0];
   // return the first item from the array
-};
 
-const returnLast = (arr) => {
+const returnLast = arr => arr[arr.length - 1];
   // return the last item of the array
-};
 
-const getArrayLength = (arr) => {
+const getArrayLength = arr => arr.length;
   // return the length of the array
-};
 
-const incrementByOne = (arr) => {
+const incrementByOne = arr => arr.map(i => i + 1);
   // arr is an array of integers
   // increase each integer by one
   // return the array
-};
 
 const addItemToArray = (arr, item) => {
+  arr.push(item);
+  return arr;
+};
   // add the item to the end of the array
   // return the array
-};
 
 const addItemToFront = (arr, item) => {
+  arr.unshift(item);
+  return arr;
+};
   // add the item to the front of the array
   // return the array
   // hint: use the array method .unshift
-};
 
-const wordsToSentence = (words) => {
+const wordsToSentence = words => words.join(' ');
   // words is an array of strings
   // return a string that is all of the words concatenated together
   // spaces need to be between each word
   // example: ['Hello', 'world!'] -> 'Hello world!'
-};
 
-const contains = (arr, item) => {
+const contains = (arr, item) => arr.indexOf(item) > -1;
```

Comment:
Good use of `indexOf`.  There's now an `includes` method that you can use.  And usually you see this written as `arr.indexOf(item) !== -1;` but it's all doing the same thing.